waiter/src/waiter/config.clj

```diff
@@ -0,0 +1,37 @@
+;;
+;; Copyright (c) Two Sigma Open Source, LLC
+;;
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;  http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+(ns waiter.config)
+
+(def ^:private config-promise (promise))
+
+(defn initialize-config
+  "Initializes the config."
```

Comment:
I think we need a more complete docstring here. Maybe explain that this is the global waiter config, as read at launch time, after post-processing. The configuration is constant for a given Waiter instantiation/process.

Reply:
Updated the docstring. Let me know whether you like it.